2-python-fundamentals-i.ipynb

```diff
@@ -4,7 +4,7 @@
    "cell_type": "markdown",
```

Comment:
_Commented on notebook `2-python-fundamentals-i.ipynb` Cell 1 Line 3_
```markdown
# 2. Python Fundamentals I: Data Types,  Data Structures, Control Flow

1. Data types
```
This outline has been helpful for my review! But it can go before merging.

Maybe add a sentence setting the context for this module?

Reply:
Actually maybe just move it to a subtitle

```
# 2. Python Fundamentals I

Data Types,  Data Structures, Control Flow
```

That'll also keep our TOC more clear:

![image](https://github.com/user-attachments/assets/596c7b9c-3c34-4142-bba5-8e66ae612d68)